libopenage/CMakeLists.txt

```diff
@@ -255,7 +247,10 @@ endif()
 ##################################################
 # directories for header inclusion
 target_include_directories(libopenage
+	PUBLIC
+	${CMAKE_CURRENT_BINARY_DIR}
 	PRIVATE
+	${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
not sure if that is a good idea, i mean the inclusion of the source-dir.

Reply:
The generated files cannot reference the header files using relative paths.
This is actually common practice, until we separate the "public" headers in a directory like `include`, which can be added by name (implicitly referencing the source directory).